### DISCUSSION TITLE
Same sex families

### DISCUSSION THREAD

**Discussion by andye2801** [gramps-project/gramps-web#519] [Ideas]
Hi,

Can it be made possible to add two mothers to a family?

**Comment by DavidMStraub** [comment]
Hi,

it is already possible. You can check the [Gramps forum](https://gramps.discourse.group/) where there are several threads about the topic.

Some wording in the interface (father/mother) will be incorrect, but I think that's an acceptable trade-off which is similar to how any two people with offspring, or any two people in a relationship, are called "Family" in Gramps, whether it's a social family or not.

I'll convert this to a discussion.

**Reply by DavidMStraub** [reply]
To be clear, by "it is already possible" I meant that you can add two people with the same sex as "father" and "mother" of a family.

**Comment by ElDubsNZ** [comment]
The gramps app has updated their terminology to now be "Father/partner1" and "Mother/partner2", it would be great if this were reflected in Gramps web to recognise the field is for any type of relationship.

**Reply by DavidMStraub** [reply]
In my opinion it would be good to use father/partner 1 and mother/partner 2 in the form where the family is created, but derive father/mother vs partner 1/partner 2 from the partner's gender in the family view. This would make it more specific.
